generators/server/templates/build.gradle.ejs

```diff
@@ -586,9 +585,9 @@ wrapper {
 
 if (project.hasProperty("nodeInstall")) {
     node {
-        version = "${node_version}"
-        npmVersion = "${npm_version}"
-        yarnVersion = "${yarn_version}"
+        version = "$<%= NODE_VERSION %>"
```

Comment:
It looks like the issue might be this `$`

Reply:
Or at least... "an" issue ;)

Reply:
:facepalm: